Ahmedq02/task-app
# Postgres migration

# PostgreSQL Migration



## Description
Migrated from SQLLite to PostgresSQL.

- **Related Issue:** Closes #10 
- **Summary of Changes:**
  - Change Postgres as database engine in app
  - Add new database environment variables
  - Refactor docker compose
  - Fix issues with `entropoint.sh` (creates the passkey automatically)

## How to Test



1. Check out this branch: `git checkout postgres-migration`
2. Complete all the environment variables described in `.env.dev`
3. Run the development server: `docker compose up --build`
4. Test the app runs

## Deployment Notes



- [ ] This PR requires database migrations
- [x] New environment variables need to be added (all new variables are described in `.env.dev`)
